test/client/end-to-end.clienttest.js

```diff
@@ -10,9 +9,9 @@ module.exports = {
       .assert.urlEquals('http://localhost:8000/search?q=ada')
       .click('.resultcard')
       .pause(1000)
-      .assert.urlEquals('http://localhost:8000/people/cp38764/ada-lovelace')
+      .assert.urlEquals('http://localhost:8000/objects/co8228143/fort-ada-alexandria-photograph')
```

Comment:
I'm not sure if this test should be changed or if we should check the search boost to know why Ada is not the first result. @Danwhy any thought or do you think this result is normal?